### PR TITLE
Make it possible to change the default color of DepthMap visualization.

### PR DESCRIPTION
**Pull request by chhtz**
Furthermore, make it possible to combine colorization and remission visualization.
Also, fixed some typos.

### CODE REVIEW

File: viz/DepthMapVisualization.cpp [hunk 1]
```diff
@@ -98,10 +100,15 @@ void DepthMapVisualization::updateMainNode ( osg::Node* node )
     }
     else if(show_remission && !scan_sample.remissions.empty())
     {
-        for(unsigned i = 0; i < scan_sample.remissions.size(); i++)
+        for(unsigned i = 0; i < points.size(); i++)
         {
 	    if(scan_sample.isIndexValid(i))
```
Comment (doudou) [inline viz/DepthMapVisualization.cpp]: If you change the index from remissions to points, you probably should validate that points.size() == remissions.size().

Comment (chhtz) [inline viz/DepthMapVisualization.cpp]: Done.

File: viz/DepthMapVisualization.cpp [hunk 2]
```diff
@@ -113,7 +124,7 @@ void DepthMapVisualization::updateMainNode ( osg::Node* node )
     }
     else
     {
-        colors->push_back(osg::Vec4(1.0,0,0.3,0.8));
+        colors->push_back(default_feature_color);
```
Comment (doudou) [inline viz/DepthMapVisualization.cpp]: Wouldn't it make more sense to keep the default feature color to its current value ? Not sure if/what you discussed with the primary users of this visualization ...

Comment (chhtz) [inline viz/DepthMapVisualization.cpp]: I changed the default to the old color. That change was honestly merely for my convenience.

**Pull request merged**